pkg/packagekit/updates.jsx

```diff
@@ -1248,14 +1233,14 @@ class OsUpdates extends React.Component {
                     else if (always_load)
                         this.loadUpdates();
                 })
-                .fail(this.handleLoadError);
+                .catch(this.handleLoadError);
     }
 
     watchUpdates(transactionPath) {
         this.setState({ state: "applying", applyTransaction: transactionPath, allowCancel: false });
 
         PK.call(transactionPath, "DBus.Properties", "Get", [PK.transactionInterface, "AllowCancel"])
-                .done(reply => this.setState({ allowCancel: reply[0].v }));
+                .then(([prop]) => this.setState({ allowCancel: prop.v }));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17517-20220705-092548-cbb7d34f-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)